ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -134,3 +134,30 @@ spec:
 #            claim: "image-registry-pvc"
 #     ---- sources needed for image registry ends here ----
 
+#    --- sources needed for updating CRI-O workload-partitioning ----
+#    more info here: on the base64 content https://docs.openshift.com/container-platform/4.11/scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.html
+#    - fileName: MachineConfigGeneric.yaml
+#      policyName: "config-policy"
+#      remediationAction: enforce # This is to update array entry as opposed to appending a new entry.
```

Comment:
In production the Policy should be generated with "inform" remediationAction and TALM used to roll it out to the clusters in the fleet. It is valid/ok to create an enforce policy for lab use/single cluster management/etc but our reference should align with the production use case.